src/App.js

```diff
@@ -2,15 +2,18 @@ import "./App.css";
 import "bootstrap/dist/css/bootstrap.min.css";
 import ShoppingCartTable from "./ShoppingCartPage/ShoppingCartTable";
 import FrontPage from "./FrontPage";
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import Header from "Header/Header";
 
 function App() {
   return (
     <Router>
       <div className="App">
-        <header className="App-header">{/* cia eis bar */}</header>
-        <Route exact path="/" component={FrontPage}/>
-        <Route exact path="/cart" component={ShoppingCartTable}/>
+        <header className="App-header">
```

Comment:
Headeriu inception 😄 Gal palikti čia tik Header ir padaryt kad pats komponentas returnintu ne tik Container o ir šitą headeri? Ar dar kažkas eis į šitą be Header?

Reply:
![image](https://user-images.githubusercontent.com/57706951/111782144-5ae5b000-88c1-11eb-8611-367bb0d07137.png)
done :D